public-ui/src/app/components/db-no-results/no-results-search.component.tsx

```diff
@@ -0,0 +1,188 @@
+import { Component, Input } from '@angular/core';
+import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
+import { domainToRoute, surveyIdToRoute } from 'app/utils/constants';
+import { reactStyles } from 'app/utils/index';
```

Comment:
Nit: import path can be shortened:
```suggestion
import { reactStyles } from 'app/utils';
```

Reply:
Thanks changed this.